EIPS/eip-5503.md

```diff
@@ -0,0 +1,176 @@
+---
+eip: 5503
+title: Refundable Token Standard
+author: <admin@startfund.io>
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-08-16
+---
+
+## Abstract
+The value of a token can be the total sum of the linked currency’s value. For example, in the Token issuing process, the issuer can receive money from buyers( or investors) and transfer issuing token to buyers. . If the offering process is completed, there is no issue. But buyers can change their plan, or the offering is failed(or be canceled) cause of misfitting the compliance rules or other rules. There is no way guarantee to pay back (refund) to the buyer in the on-chain network.
+We have suggested this process make possible in on-chain network with a payable currency like token(ex: USDT)
+
+## Motivation
+A standard interface allows the payable token contract to interact with ERC-2000 interface within smart contracts.
+
+Any payable token contract call ERC-2000 interface to exchange with issuing token based on constraint built in ERC-2000 smart contract to validate transactions.
+
+Note: Refund is only available in certain conditions(ex: period, oracle value, etc) based on implementations.
+
+## Requirements
```

Comment:
[editorial] "Requirements" are not a standard section, please consider follow the template. If you choose to create such section, please consider differentiate "requirements" vs "specification" section as it seems to me a bit confusing when I read as a reader.

Reply:
Remove Requirement section.